apps/wing-console/console/server/src/index.ts

```diff
@@ -216,8 +184,7 @@ export const createConsoleServer = async ({
   simulator.on("started", () => {
     appState = "success";
 
-    // Clear tests when simulator is restarted
-    testsStateManager().setTests([]);
+    testRunner.initialize();
```

Comment:
Is it necessary to initialize the test runner after the simulator starts?

Reply:
Yes if we want to restart the tests when there is a change in the wingfile (that causes a restart on the simulator)